src/pages/community/members.js

```diff
@@ -1,26 +1,60 @@
 import React, { useState } from "react";
-import { ThemeProvider } from "styled-components";
+import styled, { ThemeProvider } from "styled-components";
 import theme from "../../theme/app/themeStyles";
 import { GlobalStyle } from "../../sections/app.style";
 import SEO from "../../components/seo";
 import Navigation from "../../sections/General/Navigation";
 import Footer from "../../sections/General/Footer";
 import Layout from "../../components/layout";
+import MembersGrid from "../../sections/Community/Members-grid/index";
+
+import Community from "../../sections/Community/Members-grid/community";
+import Landscape from "../../sections/Community/Members-grid/landscape";
+import ImageHub from "../../sections/Community/Members-grid/imagehub";
+import Meshery from "../../sections/Community/Members-grid/meshery";
+import MesheryOperator from "../../sections/Community/Members-grid/mesheryoperator.js";
+import SMP from "../../sections/Community/Members-grid/smp";
+import GetNighthawk from "../../sections/Community/Members-grid/getnighthawk";
+
 import AllMembers from "../../sections/Community/Members-grid/AllMembers";
 import ActiveMembers from "../../sections/Community/Members-grid/ActiveMembers";
 import InactiveMembers from "../../sections/Community/Members-grid/InactiveMembers";
 
 import Maintainers from "../../sections/Community/Members-grid/Maintainers";
 import Meshmate from "../../sections/Community/Members-grid/Meshmate";
+import { bool } from "prop-types";
 
 const options = [
-  { label: "All Members", value: "all", color: `${theme.linkColor}`, isFixed: true},
-  { label: "Active Members", value: "active", color: `${theme.linkColor}`, isFixed: true},
-  { label: "Inactive Members", value: "inactive", color: `${theme.menuColor}`, isFixed: true},
-  { label: "Maintainers", value: "maintainers", color: `${theme.linkColor}`, isFixed: true},
-  { label: "MeshMates", value: "meshmates", color: `${theme.linkColor}`, isFixed: true}
+  { label: <option disabled className="category">BADGES&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;</option>, value: "", color: `${theme.linkColor}`,isFixed: true},
```

Comment:
These spaces need to be applied via css as it may not be uniform in all devices if done this way